agent/liquidity_buffer_service.py

```
@@ -0,0 +1,287 @@
+        
+        Raises:
+            ValueError: If buffer has insufficient balance
+        """
+        session = self.db.get_session()
+        try:
+            buffer = session.query(LiquidityBuffer).filter(
+                LiquidityBuffer.buffer_id == self._buffer_id
+            ).first()
+            
+            if not buffer:
+                raise ValueError("Liquidity buffer not found")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)